manual/core/ssl/README.md

```diff
@@ -101,22 +114,21 @@ passwords:
 
 If you need more control than what system properties allow, you need to write your own engine
 factory. If you just need specific configuration on the `SSLEngine`, you can extend the default
-factory and override `newSslEngine`. For example, here is how to enable hostname verification:
+factory and override `newSslEngine`. For example, here is how you would configure custom
+`AlgorithmConstraints`:
```

Comment:
This is a bit of a lame example, couldn't think of anything else a user would want to configure.

Reply:
That should be clear enough.